provision/kernel.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
```

Comment:
Please add comments describing what this is doing - context is key for maintaining code like this.